scripts/runtime-dep.py

```diff
@@ -0,0 +1,32 @@
+# To run this script, you need to install the 'toml' python package and install the 'graphviz' package:
```

Comment:
You should add `#!/usr/bin/env python3` to the top and make the script executable. 
Then you don't need to call it with `python ./scripts/runtime-dep.py`.

Reply:
I want to user to specify their own version of python. ;)  Kidding, fixed now.